cmd/kubeadm/app/preflight/checks.go

```diff
@@ -878,6 +878,8 @@ func RunInitMasterChecks(execer utilsexec.Interface, cfg *kubeadmapi.InitConfigu
 		PortOpenCheck{port: int(cfg.LocalAPIEndpoint.BindPort)},
 		PortOpenCheck{port: 10251},
```

Comment:
PortOpenCheck for 10251 and 10252 is still necessary?

Reply:
The ports are still accessible (even when unused), at least in my 13.0 testing.

I'm unsure whether kcm and scheduler fail if these ports can't be opened. I'll test that later this week.